cmake/mirrors/gitcode.cmake

```diff
@@ -0,0 +1,69 @@
+# Tengine (Download via commit id)
+ocv_update(TENGINE_GITCODE_PKG_MD5 1b5908632b557275cd6e85b0c03f9690)
+ocv_update(TENGINE_GITHUB_PKG_MD5 23f61ebb1dd419f1207d8876496289c5) # same as tengine_md5sum for TENGINE commit of e89cf8870de2ff0a80cfe626c0b52b2a16fb302e
```

Comment:
> `TENGINE_GITHUB_PKG_MD5`
> `_GITHUB_`

GitHub is misleading here. By design OpenCV may use any servers to download original external code.
Some time ago we used sourceforge.net for IPPICV package.

Name should be `TENGINE_PKG_ORIGINAL_MD5`

Reply:
Source is added as suffix to each package-related vars, such as `TENGINE_PKG_MD5_ORIGINAL`.